doc/src/suite-design-guide/style-guide.tex

```diff
@@ -132,8 +138,8 @@ \subsubsection{Script String Lines}
                  """
 \end{lstlisting}
 
-(The leading whitespace would end up in \lstinline=log.txt=, and indenting the
-\lstinline=_EOF_= marker is actually an error).
+(The leading whitespace would end up in \lstinline=log.txt=,
+and there may be other unwanted surprises).
```

Comment:
Again, not true on this branch - `dedent` removes the leading whitespace and leaves a valid heredoc in the job script.

Reply:
So the comment in the example "# .... this is an error!" should be removed too.